codebase/Dossier/src/components/HospitalNumber.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { Component, useState } from 'react';
+import { Text, TextInput, View } from 'react-native';
```

Comment:
*nit*: `Text` isnt used in this Component (being marked as an unused import my end)

Reply:
Nice spot, it also said that useState wasn't being used, so I've removed that